.gitignore

```diff
@@ -25,3 +25,4 @@ __pycache__
 .turbo/
 .cursorrules
 .corepack
+justfile
```

Comment:
Hmm what is this?  Is it intentional?

Reply:
Yeah, justfile is like a Makefile (but with a simplified syntax) it's a local command runner I use in terminal which probably isn't helpful to commit

Reply:
totally reasonable, feel free to commit files like this if you think they may be useful for others, for instance I've committed my nix files (flake.nix etc) -- maybe in a separate commit though